lib/index.js

```diff
@@ -4,6 +4,16 @@ var util = require('util');
 var EventEmitter = require('events').EventEmitter;
 
 /**
+ * routine errors
+ */
+var er = {
+  op              : 'nanomsg operation failed: ',
+  param           : 'parameter type err: ',
+  chan            : 'setting channel subscription string: ',
+  rmchan          : 'removing channel subscription string: ',
+}
```

Comment:
There's really no need for this, just put the strings in place.  DRY has its place, but not with error message substrings.